v03_pipeline/lib/misc/sample_ids_test.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+
+import hail as hl
+from hail import Struct
```

Comment:
this stands out to me as being off style-wise... I've always seen `hl.Struct` used. Though I don't really know if there's actually codified style rules or just conventions 🤔.

Reply:
good catch I'm using `hl.Struct` instead now